storage/innobase/row/row0import.cc

```diff
@@ -3073,6 +3126,149 @@ static size_t get_buf_size()
       ;
 }
 
+/** Add fts index to the table
+@param table fts index to be added on the table */
+static void add_fts_index(dict_table_t *table)
+{
+  dict_index_t *fts_index= dict_mem_index_create(
+    table, FTS_DOC_ID_INDEX_NAME, DICT_UNIQUE, 2);
+  dict_hdr_get_new_id(NULL, &fts_index->id, NULL);
+  fts_index->page = FIL_NULL;
+  fts_index->cached = 1;
+  /* Add fields for FTS_DOC_ID_INDEX */
+  dict_index_add_col(
+    fts_index, table,
+    &table->cols[table->n_cols - (DATA_N_SYS_COLS + 1)], 0);
+  dict_index_t *clust_index= UT_LIST_GET_FIRST(table->indexes);
+  for (ulint i= 0; i < clust_index->n_uniq; i++)
+    dict_index_add_col(fts_index, table, clust_index->fields[i].col,
+                       clust_index->fields[i].prefix_len);
+  fts_index->search_info= btr_search_info_create(fts_index->heap);
+  fts_index->search_info->ref_count= 0;
```

Comment:
This would break the `cmake -DWITH_INNODB_AHI=OFF` build. Please add
```c++
#ifdef BTR_CUR_HASH_ADAPT
#endif
```
around any `search_info` references.